POLICIES.rdoc

```diff
@@ -1,3 +1,16 @@
+== Pull Requests
+
+Contributions to RubyGems are made via GitHub pull requests, which must be
+approved by a project committer other than the author. To approve a PR, a
+maintainer can leave a comment including the text "@homu r+", indicating that
+they have reviewed the PR and approve it. [Homu](http://homu.io) will then
+automatically create a merge commit, test the merge, and land the PR if the
+merge commit passes the tests.
+
+This process guarantees that our release branches are always be green, and
```

Comment:
It'd be clearer to explicitly say what a branch being "green" means.

E.g., "This process guarantees that our release branches never have failing tests, [...]"